### ISSUE TITLE
Error when generating treks KML

### ISSUE THREAD

**Issue by camillemonchicourt** [makinacorpus/Geotrek#1513]
After having an error on this very long trek - https://github.com/makinacorpus/Geotrek/issues/1512
I tried to download KML of shorter treks. 
I also gets an Internal Error on all other treks but not the same, even on very short treks : 

`RuntimeError('maximum recursion depth exceeded while calling a Python object',)`

And error details is a very long : 

```
File "/home/sentiers/Geotrek/lib/eggs/Django-1.6.5-py2.7.egg/django/core/handlers/base.py", line 112, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)

  File "/home/sentiers/Geotrek/lib/eggs/Django-1.6.5-py2.7.egg/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)

  File "/home/sentiers/Geotrek/lib/eggs/Django-1.6.5-py2.7.egg/django/utils/decorators.py", line 29, in _wrapper
    return bound_func(*args, **kwargs)

  File "/home/xxxxxx/Geotrek/lib/eggs/Django-1.6.5-py2.7.egg/django/contrib/auth/decorators.py", line 22, in _wrapped_view
    return view_func(request, *args, **kwargs)

  File "/home/xxxxxx/Geotrek/lib/eggs/Django-1.6.5-py2.7.egg/django/utils/decorators.py", line 25, in bound_func
    return func(self, *args2, **kwargs2)

  File "/home/xxxxxx/Geotrek/geotrek/trekking/views.py", line 113, in dispatch
    return super(TrekKMLDetail, self).dispatch(*args, **kwargs)

  File "/home/xxxxxx/Geotrek/lib/eggs/mapentity-1.16.0-py2.7.egg/mapentity/views/mixins.py", line 55, in dispatch
    return _dispatch(*args, **kwargs)

  File "/home/xxxxxx/Geotrek/lib/eggs/Django-1.6.5-py2.7.egg/django/views/decorators/http.py", line 147, in inner
    response = func(request, *args, **kwargs)

  File "/home/xxxxxx/Geotrek/lib/eggs/mapentity-1.16.0-py2.7.egg/mapentity/views/mixins.py", line 54, in _dispatch
    return super(LastModifiedMixin, self).dispatch(*args, **kwargs)

  File "/home/xxxxxx/Geotrek/lib/eggs/Django-1.6.5-py2.7.egg/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)

  File "/home/xxxxxx/Geotrek/lib/eggs/Django-1.6.5-py2.7.egg/django/views/generic/detail.py", line 112, in get
    return self.render_to_response(context)

  File "/home/xxxxxx/Geotrek/geotrek/trekking/views.py", line 117, in render_to_response
    response = HttpResponse(trek.kml(),

  File "/home/xxxxxx/Geotrek/geotrek/trekking/models.py", line 180, in kml
    return kml._genkml()

  File "build/bdist.linux-x86_64/egg/simplekml/kml.py", line 211, in _genkml
    return kml_str.toprettyxml(indent="    ", newl="\n", encoding="UTF-8").decode("utf-8")

  File "/usr/lib/python2.7/xml/dom/minidom.py", line 57, in toprettyxml
    self.writexml(writer, "", indent, newl, encoding)

  File "/usr/lib/python2.7/xml/dom/minidom.py", line 1751, in writexml
    node.writexml(writer, indent, addindent, newl)

  File "build/bdist.linux-x86_64/egg/simplekml/base.py", line 287, in writexml
    KmlElement._original_element.writexml(self, writer, indent, addindent, newl)

  File "build/bdist.linux-x86_64/egg/simplekml/base.py", line 287, in writexml
    KmlElement._original_element.writexml(self, writer, indent, addindent, newl)

  File "build/bdist.linux-x86_64/egg/simplekml/base.py", line 287, in writexml
    KmlElement._original_element.writexml(self, writer, indent, addindent, newl)

  File "build/bdist.linux-x86_64/egg/simplekml/base.py", line 287, in writexml
    KmlElement._original_element.writexml(self, writer, indent, addindent, newl)

....
....

```

**Comment by camillemonchicourt** [comment]
Got the same error on some treks when synchronizing from Geotrek-admin 2.4.2 : 

![kml-maximum](https://cloud.githubusercontent.com/assets/4418840/11430221/94968366-9483-11e5-86f0-85d0744d6d57.jpg)

**Comment by camillemonchicourt** [comment]
Seems to be fixed in 2.10.4 with https://github.com/makinacorpus/Geotrek/commit/1897122239c8b7e9070a9be53e0bd3e88262e38c

Reopen if it happens again.
